bottom-line/src/main/resources/runTransEthnic.sh

```diff
@@ -38,19 +37,25 @@ fi
 
 # for each ancestry, merge all the results into a single file
 for ANCESTRY in "${ANCESTRIES[@]}"; do
-    GLOB="${SRCDIR}/ancestry=${ANCESTRY}/part-*"
+    GLOB="${SRCDIR}/ancestry=${ANCESTRY}/"
     ANCESTRY_DIR="${OUTDIR}/ancestry=${ANCESTRY}"
     JSON_FILE="${ANCESTRY_DIR}/variants.json"
     CSV_FILE="${ANCESTRY_DIR}/variants.csv"
 
     # create the destination directory and merge variants there
+    sudo mkdir -p tmp_files
     sudo mkdir -p "${ANCESTRY_DIR}"
-    sudo chmod 777 $ANCESTRY_DIR
-    hadoop fs -getmerge -nl -skip-empty-file "${GLOB}" "${JSON_FILE}"
+    sudo chmod 777 "${ANCESTRY_DIR}"
+
+    sudo aws s3 cp "${GLOB}" ./tmp_files/ --recursive
+    sudo zstd -d --rm ./tmp_files/part-*
+    cat ./tmp_files/part-* > "${JSON_FILE}"
```

Comment:
Probably not worth it if you've already tested things, but I think you should be able to collapse these two lines into one with zcat.  `zcat ./tmp_files/part-* > ${JSON_FILE}`